include/bx/thread.h

```diff
@@ -157,12 +165,10 @@ namespace bx
 		{
 #if BX_PLATFORM_OSX || BX_PLATFORM_IOS
 			pthread_setname_np(_name);
```

Comment:
Can you move just implementation of pthread_setname_np into compatibility headers?